### DISCUSSION TITLE
Icon Request: fa-video-card

### DISCUSSION THREAD

**Discussion by syromiatnikovv** [FortAwesome/Font-Awesome#21167] [Icon Requests]
video card PC
![stock-vector-computer-components-icon-set-processor-motherboard-ram-video-card-hdd-ssd-sshd-power-unit-303113456](https://cloud.githubusercontent.com/assets/20704692/17216071/c7f50032-54e7-11e6-9fcd-3dae4a86eaf9.jpg)
![zestaw-ikon-podzespolow-komputerowych 1](https://cloud.githubusercontent.com/assets/20704692/17216074/cc3e4798-54e7-11e6-8854-58acff08bef6.jpg)

**Comment by aethior** [comment]
Duplicate of #12990?

cc/ @tagliala

**Comment by tagliala** [comment]
Actually the example is a duplicate but video card and GPU are different.

I can't tell

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
